docs/app/Examples/elements/Divider/Types/DividerVerticalExample.js

```diff
@@ -0,0 +1,28 @@
+import React, {Component} from 'react';
+import {Segment, Divider} from 'stardust';
+
+export default class DividerVerticalExample extends Component {
+  render() {
+    return (
+      <div className='ui three column relaxed grid'>
+        <div className='column'>
+          <Segment className='basic'>
+            Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Donec odio.
+          </Segment>
+        </div>
+        <Divider className='vertical'>Or</Divider>
+        <div className='column'>
+          <Segment className='basic'>
+            Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Donec odio.
+          </Segment>
+        </div>
+        <Divider className='vertical'>And</Divider>
+        <div className='column'>
+          <Segment className='basic'>
+            Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Donec odio.
+          </Segment>
+        </div>
+      </div>
+    );
+  }
+}
```

Comment:
Oops, comments on this above before seeing your comment.  Yep, just add `Grid, Column` to the import from stardust.  Like other components, they include all the necessary base classes so you only need to add modifier classes (like `three column relaxed` to the `<Grid />`).